### PR TITLE
Implement a tickrate configurable in the command line

### PR DESCRIPTION
**Pull request by ghost**
The default tick rate is now 20Hz (0.05 s) instead of 60Hz.

I chose 20Hz because it looked like the smallest value that provided good enough user experience without client-side prediction.

I also fixed a few inconsistencies in how the update code got the tick duration.

### CODE REVIEW

**Comment by Nemikolh** [comment]
Nice! :tada: 

One nit though, could you make the unit of the command line argument to be ms instead? It's just my preference, so feel free to ignore it.

**Comment by lthms** [comment]
I also feel more natural to have the tick duration rather than the tick frequency

**Comment by ghost** [comment]
@Nemikolh I can, but I want to keep the internal units in s / tick (so directly divide the value by 1000 after parsing the command line)

**Comment by Nemikolh** [comment]
> I can, but I want to keep the internal units in s / tick (so directly divide the value by 1000 after parsing the command line)

@Vaelden Huh.. I think you meant to multiply the value? XD

It's just my preference anyway, if you prefer to have seconds on the command line, fair enough. You'll be the one using it more than anyone else so. :)

**Comment by ghost** [comment]
OK I switched the command line unit to ms, I think this can now be merged

File: src/entity/update/movement.rs [hunk 1]
```diff
@@ -36,7 +37,7 @@ fn resolve_collisions(
         Vector2::new(0.0, 0.0)
     };
     let speed = unitary_speed * entity.stats.speed;
-    let new_position = entity.position + (speed * *SEC_PER_UPDATE);
+    let new_position = entity.position + speed * tick_duration;
```
Comment (Nemikolh) [inline src/entity/update/movement.rs:40]: Oh this is surprising. Typically the physics use a separate time step. (typically as in unity) :)

**Review by Nemikolh — APPROVED** [review]

**Comment by Nemikolh** [comment]
LGTM

File: src/entity/update/movement.rs (hunk 1 above)
Comment (ghost) [reply src/entity/update/movement.rs:40]: And we may need to do that later on, if it solves a problem we encounter ...

**Pull request merged**